frontend/src/app/AboutDialog.scss

```diff
@@ -0,0 +1,9 @@
+// TODO: Remove when PF provides a fix for https://github.com/patternfly/patternfly/issues/6871
```

Comment:
Logged https://issues.redhat.com/browse/RHOAIENG-9742 to track this on the flip side.